### ISSUE TITLE
Adding Spek test to configuration of another Spek test is not working in AS 3.5

### ISSUE THREAD

**Issue by ispbox** [JetBrains/spek#778]
The story started here: https://github.com/spekframework/spek/issues/773

Issue is with running one Spek test from another one via configuration option: 
![Attach another configuration](https://user-images.githubusercontent.com/16974448/64080014-c2e09f80-ccf7-11e9-8fc6-13449e5c60fa.png)

I've created small sample project, please take a look.
[SpekTest.zip](https://github.com/spekframework/spek/files/3563756/SpekTest.zip)

If you run SampleSpekTest - it works pretty fine. But RunAllSpekTest is not working, because it has another configuration added. If you remove it, and restart Android Studio - this second test will start work.

BTW, after failure to run RunAllSpekTest you will not be able to run even SampleSpekTest until restart of AS!

**Comment by ispbox** [comment]
The issue is still there with 2.0.7.

**Comment by raniejade** [comment]
Yes, I haven't looked at this yet. In the meantime can you just use a gradle task to run everything?

**Comment by ispbox** [comment]
@raniejade Do you mean something like
```./gradlew testVariantNameUnitTest```?

My experience with that some time ago finished with disability to run multi-module tests in single task. Probably that's easy, but I didn't find any way.
Currently I just run tests by hand, one after one.

**Comment by raniejade** [comment]
> Do you mean something like
./gradlew testVariantNameUnitTest?

Yes

> My experience with that some time ago finished with disability to run multi-module tests in single task. 

You mean, it does not work anymore? 

(sorry for the delayed reply)

**Comment by ispbox** [comment]
I failed to make it working for multiple modules. In other words, it never worked for me. Maybe will do one more attempt to learn how to make this working via Gradle.

**Comment by guliash** [comment]
Hello @ispbox. Any success with resolving the issue?

**Comment by guliash** [comment]
https://issuetracker.google.com/issues/144443063
Opened issue.

**Comment by ispbox** [comment]
@guliash No, unfortunately I didn't have enough time to look at this issue. Tried right now, and the issue is still there with Spek 2.0.8.
Thanks for opening issue.

**Comment by guliash** [comment]
This should be resolved in Android Studio 4.0.
